openslides/core/static/js/core/base.js

```diff
@@ -953,6 +948,33 @@ angular.module('OpenSlidesApp.core', [
     }
 ])
 
+/* Converts a snake-case string to CamelCase. */
```

Comment:
The comment is not completely correct: `foo-bar` is converted to `fooBar` and not to `FooBar`. May be add a correct example to show what is going on here.